src/main/java/de/kaleidox/e2uClaim/E2UClaim.java

```diff
@@ -145,6 +147,19 @@ public static FileConfiguration getConfig(String name) {
         });
     }
 
+    public static void set(String fileName, FileConfiguration configuration) throws IOException {
```

Comment:
there is no need to replace a configuration; those objects are uniquely bound to their names